api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/organizations_controller.rb

```diff
@@ -290,8 +285,12 @@ def support_enabled?
     return head :forbidden unless Settings.admin_panel.support.enabled && current_user.support?
   end
 
-  def support_org_external_id
-    params[:organization_external_id]
+  def support_org_search
+    # Support users can only search organizations by external id or name.
+    external_id = params.dig(:where, :external_id)
+    name = params[:terms].present? && JSON.parse(params[:terms])["name.like"]
+
+    external_id || name
```

Comment:
I think should delete all other params/terms when the user is support?

Either use `Hash#slice` or Strong Parameters

Reply:
@ouranos yeah I like this.